spec/home.spec.tsx

```diff
@@ -1,24 +1,53 @@
-import { test, expect } from "vitest"
+import { test, expect, vi } from "vitest"
 import { render, screen } from "@testing-library/react"
 import { NostoProvider, NostoHome, NostoPlacement } from "../src/index"
 import RecommendationComponent from "./renderer"
 import { waitForRecommendations } from "./utils"
+import { jsonMockDataForPage } from "./mocks"
 
 test("Home page render", async () => {
+  const actionMock = {
+    load: () => Promise.resolve(jsonMockDataForPage("front")),
+    setPlacements: vi.fn()
+  }
+
+  const sessionMock = {
+    setVariation: this,
+    setResponseMode: vi.fn(),
+    viewFrontPage: () => actionMock
+  }
+
+  const mockApi = {
```

Comment:
looks like the parameters for this mock are
* placements: ["frontpage-nosto-1", "frontpage-nosto-2"]
* action results: jsonMockDataForPage("front")